packages/hydrogen/src/customer-privacy/ShopifyCustomerPrivacy.tsx

```diff
@@ -145,36 +148,51 @@ export function useCustomerPrivacy(props: CustomerPrivacyApiProps) {
 
   useEffect(() => {
     if (scriptStatus !== 'done' || loadedEvent.current) return;
-
     loadedEvent.current = true;
 
+    if (!consentConfig.checkoutDomain) logMissingConfig('checkoutDomain');
+    if (!consentConfig.storefrontAccessToken)
+      logMissingConfig('storefrontAccessToken');
+
+    // validate that the storefront access token is not a server API token
+    if (
+      consentConfig.storefrontAccessToken.startsWith('shpat_') ||
+      consentConfig.storefrontAccessToken.length !== 32
+    ) {
+      logInvalidConfig('storefrontAccessToken');
+    }
```

Comment:
Would it be worthwhile instead of having a generic "invalid" property message, say something like, "it looks like you passed a private access token, make sure to use the public token". I just wonder if people would get confused, "I did pass the token, why is it erroring?"

Reply:
Agree, updated!